latch_cli/docker_utils/__init__.py

```diff
@@ -202,9 +203,12 @@ def generate_dockerfile(pkg_root: Path, outfile: Path) -> None:
             print("  - base image:", config.base_image)
             print("  - latch version:", config.latch_version)
     except FileNotFoundError as e:
-        raise RuntimeError(
-            "Could not find a .latch/config file in the supplied directory. If your workflow was created prior to release 2.13.0, you may need to run `latch init` to generate a .latch/config file."
-        ) from e
+        print(
+            "Could not find a .latch/config file in the supplied directory. Recreating configuration."
```

Comment:
```suggestion
            "Could not find an existing .latch/config file in the supplied directory. Creating configuration from scratch."
```

Reply:
I took the second half of this but "could not find an existing" seems redundant.